src/date.rs

```diff
@@ -0,0 +1,35 @@
+use std::fmt;
+use std::str::FromStr;
+
+use errors;
+
+#[derive(Debug)]
+pub struct Date(::chrono::NaiveDate);
```

Comment:
chrono has the timezone-aware `Date<Tz>`, but we can use aliases to get around it if we end up needing it.